modules/ROOT/partials/configuration/extended_valid_elements.adoc

```diff
@@ -31,8 +33,9 @@ To allow script elements in the editor, include the following in the {productnam
 extended_valid_elements : 'script[src|async|defer|type|charset]'
 ----
 
+[discrete]
 === Interactive example
 
-This example shows you how to use the link:{baseurl}/configure/content-filtering/#extended_valid_elements[extended_valid_elements] option. This option is used to add additional valid elements and attributes.
+This example shows you how to use the <<extended_valid_elements,`extended_valid_elements`>> option. This option is used to add additional valid elements and attributes.
```

Comment:
in this file, should not be preserved absolute links or transformed into something like this?
```
xref:valid_elements[valid_elements]
```